### DISCUSSION TITLE
Custom manaded properties returns null on the pnp search result

### DISCUSSION THREAD

**Discussion by tocri** [microsoft-search/pnp-modern-search#3698] [Q&A] [answered]
Hi, 
I have a SP list with a custom choice column that i named PPF_Classification. 
I would like to add the value of the column for this item here

<img width="439" alt="image" src="https://github.com/microsoft-search/pnp-modern-search/assets/10110260/73ff1e04-fabd-41fe-9541-d5b315f15bfc">

For that i have modify a refinablestring like that

![image](https://github.com/microsoft-search/pnp-modern-search/assets/10110260/31155405-0304-43d7-a3a1-3791f1024f70)

and i have added the alias to selected properties

![image](https://github.com/microsoft-search/pnp-modern-search/assets/10110260/72fdd206-9b4e-4cd0-b0f4-b120a94a31b3)

if i check in the debug layout, the query for this properties returns null
<img width="405" alt="image" src="https://github.com/microsoft-search/pnp-modern-search/assets/10110260/425470fd-3153-4fba-a7aa-4760d6408409">

My template query is : {searchTerms} Path:"{site.absoluteUrl}/Videos" Video is the name of the library. 
and in result id i have this LocalVideoResults

Do you have idea ?

**Comment by kasperbolarsen** [comment, marked as answer]
I would verify the data returned from the API using either SP Editor or the SharePoint Search Query tool.  
Sometimes it takes a looong time for the mapping to take effect .

**Reply by tocri** [reply]
You're right. the crawl was taking so much time. It's working as expected

![image](https://github.com/microsoft-search/pnp-modern-search/assets/10110260/116dc83c-389e-44d3-a831-7e3781be94e8)
